src/lib/buttons/Button.svelte

```diff
@@ -126,7 +127,9 @@
   on:keydown
   on:keyup
   on:mouseenter
-  on:mouseleave>
+  on:mouseleave
+  role={href ? undefined : "button"}
```

Comment:
if it is `<button>` do you need `role="button"`

Reply:
No you don't need it if the type is button.